src/include/cell.h

```diff
@@ -237,10 +237,20 @@ struct __wt_cell_unpack_kv {
 };
 
 /*
- * WT_CELL_UNPACK_DELTA --
- *     Unpacked delta cell.
+ * WT_CELL_UNPACK_DELTA_INT --
+ *     Unpacked internal delta cell.
  */
-struct __wt_cell_unpack_delta {
+struct __wt_cell_unpack_delta_int {
+    uint32_t __len;
+    WT_CELL_UNPACK_KV *key;
```

Comment:
I think this should be a field instead of a pointer. Same for WT_CELL_UNPACK_ADDR.